nav-app/src/app/techniques-search/techniques-search.component.scss

```diff
@@ -48,4 +67,7 @@
             text-align: center;
         }
     }
+    ::ng-deep mat-panel-description {
+        align-items: center;
+    }
```

Comment:
I believe the correct way to approach this is to disable ViewEncapsulation, ::ng-deep is deprecated:https://angular.io/guide/component-styles#deprecated-deep--and-ng-deep